notes_app/client/app/registration/registration.component.js

```diff
@@ -7,7 +7,13 @@
     ctrl.attemptRegistration = function(username, password) {
       userService.register(username, password).then(function(status) {
         ctrl.registrationState['visible'] = false;
-        ctrl.close();
+        userService.login(username, password).then(function(status) {
+          ctrl.close({'$value': true});
+        }).catch(function(status) {
+          ctrl.registrationState['message'] = 'Problem logging in; try to log in manually'
```

Comment:
I'm not sure this is the bets message. `to log in manually` - what it supposes to mean? Maybe try to login later?